src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -37,6 +37,7 @@
             <argument type="service" id="annotations.reader" />
             <argument>%kernel.cache_dir%/annotations.php</argument>
             <argument type="service" id="cache.annotations" />
+            <argument>#^Symfony\\(?:Component\\HttpKernel\\|Bundle\\FrameworkBundle\\Controller\\(?!AbstractController$|Controller$))#</argument>
```

Comment:
Can we find a way to test this regex? 🤔

Reply:
verified locally, not sure we need more, the outcome is not that important